### PR TITLE
Improvement/retries on stats metric server

### PR DESCRIPTION
**Pull request by Zempashi**

### CODE REVIEW

File: tests/utils/helper.py [hunk 1]
```diff
@@ -62,11 +63,31 @@ class RetryCountExceededError(StopIteration):
...
+            yield True
+            time.sleep(self.wait)
+        return self.msg
+
+    def __call__(self, func):
+        @functools.wraps(func)
+        def wrapper(*args, **kwargs):
+            for _ in iter(self):
+                try:
+                    return func
```
Comment (ballot-scality) [inline tests/utils/helper.py]: Call with *args, **kwargs

**Review by ballot-scality — DISMISSED** [review]
LGTM

**Review by NicolasT — DISMISSED** [review]

**Comment by Zempashi** [comment]
@bert-e wait

**Comment by Zempashi** [comment]
Superseded by #478

**Pull request closed without merging**